health/health.d/geth.conf

```
@@ -0,0 +1,25 @@
+ template: chainhead_diff_between_header_block
```

```suggestion
 template: geth_chainhead_diff_between_header_block
```

it is a common pattern to use the module name as a prefix for alarms names. This guarantees alarms names uniqueness.